### ISSUE TITLE
meta: debian: integration status

### ISSUE THREAD

**Issue by rzr** [eclipse/mosquitto#2850]
Hi,
let create this ticket to ease cooperation with debian project:

https://tracker.debian.org/pkg/mosquitto

Details to be updated...

**Comment by Nardol** [comment]
@rzr 
Nice! Some questions:
Can we expect this version to be available in Bookworm if it is then accepted in testing?
Or only a backport?
I suppose we cannot expect anything for Bullseye as it is now old stable?

Thanks for your work!

**Comment by rzr** [comment]
yes bookworm is my target , I never made a backport upload yet, let's wait it to land in testing first

For Bullseyes you can use it from my repo:

https://build.opensuse.org/package/show/home:rzrfreefr/mosquitto

**Comment by Nardol** [comment]
Nice, I use an Odroid N2+ so arm64 which looks available on your repository, thanks!
When I'll upgrade to bookworm I will continue to use your repository which also provide packages for Bookworm except if I did not understand what I read. Or could it cause some trouble when it becomes officially available on Bookworm?

**Comment by rzr** [comment]
yea feel free to use my repo meanwhile it's landing in bookworm:

```
lynx -dump 'https://ftp-master.debian.org/new/?C=M;O=A' | grep -n mosquitto | head -n 1
542:   [TXT] [488]mosquitto_2.0.15-2~bpo12+1.html 2023-07-24 07:02 196K
```

I'll share update at:

https://mastodon.social/@rzr/110768566500363383#mosquitto

**Comment by rzr** [comment]
Please review: 

https://salsa.debian.org/debian-iot-team/mosquitto/-/merge_requests/9

For latest version integration

**Comment by Nardol** [comment]
Any plan to make a backport for bookworm of the same version available in testing?

**Comment by rzr** [comment]
> Any plan to make a backport for bookworm of the same version available in testing?

not planned but you try to do it join iot team we can lead you how

**Comment by Nardol** [comment]
> not planned but you try to do it join iot team we can lead you how

Not simple as that it seams...

**Comment by rzr** [comment]
> Any plan to make a backport for bookworm of the same version available in testing?

it's in

https://packages.debian.org/source/bookworm-backports/mosquitto

thanks

**Comment by rzr** [comment]
Update: https://tracker.debian.org/news/1578594/accepted-mosquitto-2020-1-source-into-unstable/

Any help to upstream patches will not hurt:

https://salsa.debian.org/debian-iot-team/mosquitto/-/tree/debian/master/debian/patches?ref_type=heads

Relate-to: https://github.com/eclipse/mosquitto/issues/2878

**Comment by rzr** [comment]
Please @ralight or any upstream contributor you can please review this PR:
https://salsa.debian.org/debian-iot-team/mosquitto/-/merge_requests/21

The change is passing CI check, but it does not tell if we are breaking any functionality provided by v2.0.11

Relate-to: https://gitlab.eclipse.org/security/cve-assignement/-/issues/26#note_3181028
